src/main/java/com/google/cloud/genomics/dataflow/pipelines/IdentityByState.java

```diff
@@ -89,7 +89,8 @@ public static void validateOptions(Options options) {
 
   }
 
-  private static final String VARIANT_FIELDS = "variants(start,calls(genotype,callSetName))";
+  private static final String VARIANT_FIELDS =
+      "variants(alternateBases,calls(callSetName,genotype),end,referenceBases,start)";
```

Comment:
Well, a link to the specific portion of the API is probably easiest like this:

https://developers.google.com/apis-explorer/#p/genomics/v1/genomics.variants.get

Or the alternative is to have a method/class where one provides the list of fields not to include, and it will build it for you - and the parentheses and commas are automatically placed:

```Java

import java.util.*;
import java.util.stream.*;

public class FieldBuilder {
    
  public static void main( String [] args ) {

    List<String> l = Arrays.asList( "alternateBases", 
    "calls.callSetId", 
    "calls.callSetName",
    "genotype",
    "genotypeLikelihood",
    "info",
    "phaseset",
    "created",
    "end",
    "filter",
    "id",
    "info",
    "names",
    "quality",
    "referenceBases" );

    l.stream().filter( v -> !v.startsWith("created") && !v.startsWith("info") ).forEach(System.out::println);

  }
}
```

Below is the output:

> $ javac FieldBuilder.java ; java FieldBuilder
alternateBases
> calls.callSetId
> calls.callSetName
> genotype
> genotypeLikelihood
> phaseset
> end
> filter
> id
> names
> quality
> referenceBases
> $

~p